Source/WebKit/UIProcess/Cocoa/VideoFullscreenManagerProxy.mm

```diff
@@ -730,15 +732,21 @@ - (BOOL)prefersStatusBarHidden
         if (![[view layer] superlayer])
             [playerLayer addSublayer:[view layer]];
 
+        RetainPtr videoView = [[WKVideoView alloc] initWithFrame:CGRectMake(0, 0, initialSize.width(), initialSize.height())];
+        [videoView addSubview:playerView.get()];
+        [NSLayoutConstraint activateConstraints:@[
```

Comment:
Makes me a bit nervous.

Reply:
I can change this to just have the WKVideoView resize its subviews to have frames equal to its bounds.